cmd/setup-gh.go

```diff
@@ -84,6 +92,14 @@ func createGraphClient(azCred *azidentity.DefaultAzureCredential) (providers.Gra
 	return &providers.GraphServiceClient{Client: client}, nil
 }
 
+func createRoleAssignment(azCred *azidentity.DefaultAzureCredential, subscriptionId string) (providers.RoleAssignClient, error) {
```

Comment:
nit: seems like this gives back RoleAssignmentClient can you rename it to createRoleAssignmentClient()?